### ISSUE TITLE
Annotations 0.2.22

### ISSUE THREAD

**Issue by ramltribute** [raml-org/raml-js-parser-2#259]
Hello,

I'm working with the laters version of RAML parser. We hace some functionality working with toJSON() for annotations that is not working right now.

This is the annotation:

```
annotationTypes:
  bindingDefinition:
    displayName: BindingDefinition
    type: array
    items:
      properties:
        method:
          type: string
          required: true
          description: This field defines to which method the binding applies
          enum: ["GET", "PATCH", "POST", "PUT", "DELETE"]
        input:
          type: string
          required: true
          description: The binding related to the field when used as an input
          enum: ["OPTIONAL", "REQUIRED", "NONE"]
        output:
          type: string
          required: true
          description: The binding related to the field when used as an output
          enum: ["OPTIONAL", "REQUIRED", "NONE"]
```

Because allowMultiple is deprecated in RC2, we have defined it as an array. This advice was from a previous issue in repository.

So we defined it as an array, and we use it in this way:
```
types:
  cards:
    type: array
    items:
      properties:
        cardId:
          (annotations.bindingDefinition):
            - method: "GET"
              input: "NONE"
              output: "REQUIRED"
            - method: "POST"
              input: "NONE"
              output: "REQUIRED"
          type: string
          description: |
            Card identifier.
          required: false
```

The problem is that toJSON() from this datatype returns me this new object:
```
{
  "structuredValue": {
    "null": {
      "method": "POST",
      "input": "NONE",
      "output": "REQUIRED"
    }
  },
  "name": "annotations.bindingDefinition"
}
```

Previously, annotations were an array with key/values defines like the annotation. Also the null in there is strange. It seems it picks up the last member

Doing some research, i checked that API object generated by parser have the 2 items in this data type, so its not a problem of raml parsing into object, its the transformation into toJSON structure.

I can provide more information if needed

Thank you for all the work during this last month, the parser looks very good now

**Comment by KonstantinSviridov** [comment]
Thanks for your report. Now the JSON is like
```
{
	"structuredValue": [
		{
			"method": "GET",
			"input": "NONE",
			"output": "REQUIRED"
		},
		{
			"method": "POST",
			"input": "NONE",
			"output": "REQUIRED"
		}
	],
	"name": "lib.bindingDefinition"
}
```

**Comment by ddenisenko** [comment]
Fixed in 0.2.23
